abstract.md

```diff
@@ -0,0 +1,41 @@
+# Making research software more robust
+Morgan Taschuk and Greg Wilson
+
+
+At the cutting edge of research, bioinformatics software is often written by a
+single person who is both developer and scientist, producing insightful, novel
+results for their own projects. Upon presenting their work, other
+people will want to use their methods. But much of academic software is written
```

Comment:
"...for their own projects.  Other people may want to use their methods, but much of academic software..." (basically chopping out a sentence and stitching two others together)